config.py

```diff
@@ -63,7 +63,6 @@ class BaseConfig:
         'it': '@'.join(['redazione', 'cafebabel.com']),
     }
 
-    MEDIA_URL = 'https://media.cafebabel.com'
```

Comment:
How do you want to deal with the prod settings?
Should we explicitely override `MEDIA_URL = None` in some dev config?

Reply:
I think we will have a dedicated (local) config + launcher for prod that's why I prefer to remove it from globals.